src/styles/typography.js

```diff
@@ -20,6 +20,9 @@ const typography = new Typography({
   ],
   scaleRatio: 2.5,
   overrideStyles: ({ adjustFontSizeTo, rhythm }, options, styles) => ({
+    h1: {
```

Comment:
So the typography definitions are really specific in this case; Ideally these overrides are for everywhere, but most importantly there are for article/page content. A lot of these tags will be coming from the rich text editor in the CMS.

Since the header/hero h1 is very specific and breaks the scale so much it'd be better to define that h1 style within that component rather than globally as part of the typography system.

The same goes for the h6; It'd be better to define that style within the Hero/Header component (it should probably be a paragraph tag too there since that's more semantic) than globally in the typography styles. That way when an h6 is used in the CMS it follows that scale that's defined properly.

Reply:
Please see b4203b5.